server/data/data-api/src/main/java/org/apache/james/user/api/UsersRepository.java

```diff
@@ -136,5 +136,10 @@
      * @throws UsersRepositoryException
      */
     String getUser(MailAddress mailAddress) throws UsersRepositoryException;
+    
+    /**
+     * Return true if the user is an admin for this repository
+     */
+    boolean isAdministrator(String username) throws UsersRepositoryException;
```

Comment:
that's just to be coherent with the rest of this API. Can be useful for some implementations that need no throw checked exceptions